propertyestimator/client.py

```diff
@@ -125,39 +143,34 @@ class Config:
         }
 
 
-class PropertyEstimator(object):
-    """The main object responsible for requesting a set of properties
-    be calculated by the low-level property calculation backend.
+class PropertyEstimatorClient:
```

Comment:
Suggest `PropertyEstimator` instead of `PropertyEstimatorClient`, since this is the user-facing end.